comp/syntheticstestscheduler/common/data.go

```diff
@@ -0,0 +1,161 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+// Package common contains common structures and constants used for synthetics test scheduler.
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// SubType represents the network type of network test.
+type SubType string
+
+const (
+	// SubTypeUDP represents a udp network test.
+	SubTypeUDP SubType = "udp"
+	// SubTypeTCP represents a tcp network test.
+	SubTypeTCP SubType = "tcp"
+	// SubTypeICMP represents a icmp network test.
+	SubTypeICMP SubType = "icmp"
+)
```

Comment:
Shall we use the constants from: https://github.com/DataDog/datadog-traceroute/blob/main/traceroute/traceroute.go#L8-L15 

(the case is different though, maybe we should lower case the constants in datadog-traceroute for consistency)